packages/ui/src/Toggle.module.scss

```diff
@@ -0,0 +1,138 @@
+@use '../styles/constants' as c;
+@use '../styles/helpers' as h;
+
+$h: c.$grid * 6;
+$w: c.$grid * 15;
+$labelMargin: c.$grid * 4;
+$trackBr: 22px;
+$trackBgOn: #1dbc22;
+$trackBgOff: c.$colorNeutralLight;
+$knobRight: $w * 0.6;
+$knobBgOn: c.$colorNeutralWhite;
+$knobBgOff: c.$colorNeutralLighter;
+$knobR: $h * 0.9;
+$knobOffset: c.$grid * 0.3;
+$knobFontSize: c.$grid * 3.5;
+
+@keyframes fadein {
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+}
+
+@mixin hoverState {
+  border-color: c.$colorPrimary;
+}
+
+@mixin focusState {
+  @include h.outlineFormField;
+}
+
+.wrapper {
+  // hide the actual input, we will show the toggleTrack instead.
+  input[type='checkbox'] {
+    display: none;
```

Comment:
I would suggest using `@visuallyHidden` instead of `display: none` to keep it accessible for the screen readers